charts/capsule/README.md

```diff
@@ -60,6 +60,7 @@ Here the values you can override:
 
 Parameter | Description | Default
 --- | --- | ---
+`manager.hostNetwork` | Specifies if the container should be started in `hostNetwork` mode. Required for use in some managed kubernetes clusters (such as AWS EKS) with custom CNI (such as calico), because control-plane managed by AWS cannot communicate with pods' IP CIDR and admission webhooks are not working | `false` 
```

Comment:
Let's remove the following sentence:

> Required for use in some managed kubernetes clusters (such as AWS EKS) with custom CNI (such as calico), because control-plane managed by AWS cannot communicate with pods' IP CIDR and admission webhooks are not working

This has been already addressed here for further details:
https://github.com/clastix/capsule/blob/e140fb145e60f08d7ab93d9e15560ad95207cd82/charts/capsule/values.yaml#L11-L15